components/sections/featured_products/stories/featured_products.stories.tsx

```diff
@@ -0,0 +1,676 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Sections, Variants } from "types";
+import FeaturedProducts from "../index";
+
+const args = {
```

Comment:
I think for default component arguments this long maybe we could define it in a common file instead (if they're not exported from the schema packages atm). What do you think?

Reply:
Resolved